careers/_template-students-and-graduates.html

```diff
@@ -0,0 +1,9 @@
+<h2 class="h4">Students & Recent Graduates</h2>
+<p class="short-desc">
+    Find opportunities in public service for a new generation of leaders.
+</p>
+<p class="short-desc">
+    <a href="/careers/students-and-graduates/" class="jump-link jump-link__right">
+        Learn about students & graduates opportunities
```

Comment:
[Here we go.](https://github.com/cfpb/cfgov-refresh/pull/680#discussion_r33495960)

>Yes, we're using ampersands in navigation and spelling 'and' out in page titles.